src/tribol/geom/GeomUtilities.cpp

```diff
@@ -1462,15 +1480,16 @@ void PolyReorder( real* const RESTRICT x, real* const RESTRICT y, const int numP
       y[i] = ytemp[ newIDs[i] ];
    }
 
-   return;
+   return true;
 
 } // end PolyReorder()
 
 //------------------------------------------------------------------------------
-void PolyReverse( real* const RESTRICT x, real* const RESTRICT y, const int numPoints )
+TRIBOL_HOST_DEVICE void PolyReverse( RealT* const x, RealT* const y, const int numPoints )
 {
-   real xtemp[ numPoints ];
-   real ytemp[ numPoints ];
+   constexpr int max_nodes_per_elem = 4;
```

Comment:
Verify whether this should be max_nodes_per_elem or max_nodes_per_overlap. I think PolyReverse() should in general reverse the ordering of the vertices of a polygon, which leads me to believe we should use max_nodes_per_overlap, since the maximum number of vertices for a polygon generated here, at least for linear quads, is 8.

Reply:
We only call this with the vertices of an element on mesh2 and I can't think of a case where we would want to reverse the orientation of a general polygon.  Is there a reason this would be called on an intersection polygon?  Perhaps we can rename the routine so it's clear it's for an element?